packages/clerk-js/src/ui/primitives/CheckboxInput.tsx

```diff
@@ -0,0 +1,16 @@
+import type { ComponentProps } from 'react';
+import React from 'react';
+
+import { Input } from './Input';
+
+export type CheckboxInputProps = ComponentProps<typeof Input>;
+
+export const CheckboxInput = React.forwardRef<HTMLInputElement, CheckboxInputProps>((props, ref) => {
```

Comment:
Can't we avoid the extra primitive here by cloning the `Input` and passing the extra prop at `index.ts` (same for checkbox)?

Reply:
Just added `defaultProps` option for `makeCustomizable` so we can pass default props to any primitive.